cli/src/cli.rs

```diff
@@ -1549,6 +1568,22 @@ fn do_process_ether_deploy(
         config.commitment,
     )?;
 
+    {  // Create code account
```

Comment:
Why you need a separate transaction to create a code account? You can use the initial message for this purpose. See https://github.com/Cyber-Core/solana/pull/11/files#diff-ac22ed996b15b25355a0d73a66f3ce665736852cd8a7c2439a10bc2edea22000R1523

Reply:
Separate transaction - it's additional time to deploy the contract. Moreover, it is important for us to create these accounts together. If the creation of these accounts is performed in different transactions, then if the creation of the second account fails, there will be garbage in the form of the first account.